host/xtest/pkcs11_1000.c

```diff
@@ -1639,7 +1670,25 @@ struct mac_test {
 		.multiple_incr = incr			\
 	}
 
+#define CKTEST_CMAC_TEST(key, mecha, input_incr, input, output, incr) {	\
+		.attr_key = key,		\
+		.attr_count = ARRAY_SIZE(key),	\
+		.mechanism = mecha,		\
+		.in_incr = input_incr,		\
+		.in = input,				\
+		.in_len = 0,				\
+		.out = output,				\
+		.out_len = ARRAY_SIZE(output),		\
+		.multiple_incr = incr			\
```

Comment:
add a coma `,`

Reply:
Ok. Was trying to be consistent with `CKTEST_MAC_TEST`. Will add a comma there as well.